src/constants.js

```diff
@@ -724,3 +724,12 @@ export const LEADER_RECORD_STATUSES = {
 };
 
 export const USER_TOUCHED_STAFF_SUPPRESS_STORAGE_KEY = 'folio_user_touched_staff_suppress';
+
+export const COLUMN_NAMES = {
```

Comment:
- let's change the name to something more specific;
- this variable is used only in one place, maybe it shouldn't be added to general constants.

Reply:
I agree with the naming, but let's leave this variable in general constants, because it will be used in [`src/Holding/ViewHolding/HoldingAquisitions/HoldingAquisitions.js`](https://github.com/folio-org/ui-inventory/blob/e0bc47eea5b2b1c365d5ed9e534a129e61c56615/src/Holding/ViewHolding/HoldingAquisitions/HoldingAquisitions.js#L20) in scope of [UIIN-2817](https://folio-org.atlassian.net/browse/UIIN-2817)